src/buildercore/terraform.py

```diff
@@ -118,6 +126,32 @@ def render(context):
         }
         tf_file['resource'][RESOURCE_TYPE_FASTLY][RESOURCE_NAME_FASTLY]['backend']['healthcheck'] = 'default'
 
+    if context['fastly']['errors']:
+        errors = context['fastly']['errors']
+        response_objects = []
+        cache_conditions = []
+        data[DATA_TYPE_HTTP] = {}
+        for code, path in errors['codes'].items():
+            cache_condition = {
+                'name': 'condition-%s' % code,
+                'statement': 'beresp.status == %d' % code,
+                'type': 'CACHE',
+            }
+            cache_conditions.append(cache_condition)
+            response_objects.append({
+                'name': 'error-%s' % code,
+                'status': int(code),
+                'response': FASTLY_ERRORS_RESPONSE[int(code)],
```

Comment:
`http.client.responses[int(code)]`?

Reply:
Python-3 specific, but I'm looking for a portable alternative